javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var $contactForm = document.getElementById('contact-form');
+var userData = {};
```

Comment:
You should make this object in the function for the submit event, you wouldn't want to reuse the same object if the form is submitted multiple times.